handler/src/main/java/io/netty/handler/ssl/PemReader.java

```diff
@@ -149,6 +137,7 @@ private static String readContent(InputStream in) throws IOException {
             return out.toString(CharsetUtil.US_ASCII.name());
         } finally {
             safeClose(out);
+            safeClose(in);
```

Comment:
I think it's weird API to close things we didn't open. Open and close should be symmetric in the code.

Reply:
Agree with @chrisvest

Reply:
I am a bit on the fence here as well... Maybe we should just add some docs to `SslContextBuilder` and explain that the user is responsible to close the input stream ?

Reply:
Adding JavaDoc sounds much better. Because the `InputStream` is created and opened by the User so closure should be done by the User too.

Reply:
See https://github.com/netty/netty/pull/10983